scripts/bump-holochain.sh

```diff
@@ -56,18 +44,18 @@ case $CURRENT_BRANCH in
         SEARCH_PATTERN=$(echo "$VERSION_STR" | awk -F '.' 'BEGIN { OFS="" } {print "^holochain-", $1, ".", $2, ".[0-9]+$"}')
         ;;
     *)
-        echo "Not on branch main or main-X.Y, skipping for branch [${CURRENT_BRANCH}]"
+        echo "Not on branch \`main\` or \`main-X.Y\`, skipping for branch \`${CURRENT_BRANCH}\`"
         exit 0
         ;;
 esac
 
 echo "Looking for tags matching pattern: $SEARCH_PATTERN"
 
-ALL_HOLOCHAIN_TAGS=$(git -c 'versionsort.suffix=-beta-dev' -c 'versionsort.suffix=-dev' -c 'versionsort.suffix=-rc' ls-remote -q --tags --sort='v:refname' https://github.com/holochain/holochain.git | cut --delimiter='/' --fields=3)
-LATEST_MATCHING_TAG=$(echo "$ALL_HOLOCHAIN_TAGS" | grep -E "$SEARCH_PATTERN" | tail -n 1)
+ALL_HOLOCHAIN_TAGS=$(git -c 'versionsort.suffix=-dev' -c 'versionsort.suffix=-rc' ls-remote -q --tags --sort='v:refname' https://github.com/holochain/holochain.git holochain-* | cut --delimiter='/' --fields=3)
+LATEST_MATCHING_TAG=$(echo "$ALL_HOLOCHAIN_TAGS" | { grep -E "$SEARCH_PATTERN" || true; } | tail -n 1)
```

Comment:
Better error handling so that the `if` below gets a chance to run if there are no results from this match